configs/mvpose/README.md

```diff
@@ -92,7 +92,7 @@ The 2D perception data we use is generated by fasterrcnn, and you can download i
 
 | Config | Actor 0 | Actor 1 | Actor 2 | Average | Download |
```

Comment:
In addition to the detailed PCP values mentioned in the paper, shall we also report the results of other metrics selected in `pick_dict`? For MvP, MPJPE, PA-MPJPE, PCK@50, and PCK@100 are reported additionally for Shelf and Campus.